seed/seed.json

```diff
@@ -1074,6 +1074,34 @@
                 "channel": ["NIGHTLY"],
                 "platform": ["WINDOWS", "MAC", "LINUX", "ANDROID"]
             }
+        },
+        {
+            "name": "PartitionConnectionsByNetworkIsolationKeyStudy",
+            "experiments": [
+                {
+                    "name": "Enabled",
+                    "probability_weight": 100,
+                    "feature_association": {
+                        "enable_feature": [
+                            "PartitionConnectionsByNetworkIsolationKey",
+                            "PartitionDomainReliabilityByNetworkIsolationKey",
```

Comment:
@pes10k can you please confirm we need `PartitionDomainReliabilityByNetworkIsolationKey`? just not sure we have it enabled

Reply:
I think you're right, we shouldn't have the domain reliability functionality enabled (if we haven't already disabled it), and so we don't need the `PartitionDomainReliabilityByNetworkIsolationKey` feature either

Reply:
https://chromium.googlesource.com/chromium/src/+/refs/heads/main/components/domain_reliability/README.md if others aren't familiar with what the feature does (it reports to google when google properties are down)

Reply:
agree. the domain reliability feature is disabled on our end.